etc/sudoers.d/zenoss_nmap

```diff
@@ -1,4 +1,4 @@
 # Allows Zenoss to use nmap for pinging
-%zenoss ALL=(ALL) NOPASSWD: /usr/bin/nmap 
+%zenoss ALL=(ALL) NOPASSWD: /usr/bin/nmap !(*—script*) !(*-sC*)
```

Comment:
Can you explain what this does?

Reply:
Deny use -script in args line, because users can runs (/bin/bash) as nmap script file as root user